packages/experimental/NativeFontMetrics/src/NativeFontMetrics.ts

```diff
@@ -0,0 +1,19 @@
+import { ScaleFactors, TextStyle } from './NativeFontMetrics.types';
+
+interface NativeFontMetricsInterface {
+  allScaleFactors(): ScaleFactors;
+  scaleFactorForStyle(style: TextStyle): number;
+}
+
+const NativeFontMetrics: NativeFontMetricsInterface = {
+  allScaleFactors: () => {
+    console.warn('NativeFontMetrics is only available on iOS');
```

Comment:
What I've seen happen with warnings like these is that stye end up firing on all platforms. Can we quickly check that this doesn't show up on say, macOS on boot?

Reply:
It doesn't, so we're good.